### ISSUE TITLE
`runtests.py pytest` doesn't work

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#3890]
`./runtests.py -l` indicates that there is a task called `pytest` but `runtests.py pytest` doesn't run it:

```
$ ./runtests.py pytest
PARALLEL 8
SUMMARY  0 tasks selected
SUMMARY  all 0 tasks and 0 tests passed
*** OK ***
total runtime: 0.010855553999135736 sec
```

**Comment by elazarg** [comment]
Another regression of mine. What is the ideal interface? I'd suggest
`./runtests.py unit-test` runs all unit test, myunit and pytest alike.
Do we need a way to run myunit tests (or pytest) specifically?

**Comment by JukkaL** [comment]
If we name the tasks `pytest unit-test` and `legacy unit-test` then `./runtests.py unit-test` will run  both of them, because runtests uses a substring check. We can also have `pytest integration-test` for the slower tests.

**Comment by JukkaL** [comment]
Fixed by #3894.
